CPT331.Data/UserRepository.cs

```diff
@@ -66,7 +66,7 @@ public User GetUserByUsername(string username)
 			using (SqlConnection sqlConnection = SqlConnectionFactory.NewSqlConnetion())
 			{
 				user = SqlMapper
-					.Query(sqlConnection, RegistrationSpGetUserByUsername, new { EmailAddress = username }, commandType: CommandType.StoredProcedure)
```

Comment:
I think when you merged this you might have needed to choose the other side. We should be using the concreted up names rather than the magic strings.